build/preset.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Generate a Babel preset that outputs ES modules when the env is `es`, and
+ * CommonJS modules otherwise.
+ */
+
+const es = (process.env.BABEL_ENV || process.env.NODE_ENV) === 'es';
+
+module.exports = {
+  presets: [
+    ['es2015', {
+      modules: es ? false : 'commonjs',
```

Comment:
not a big fan of this, seems a bit hard to read/confusing for what is actually happening, maybe just do: 
```
if (es)  {
    module.exports = { 
        "presets": [["es2015", false], "react", "stage-0"]
    };
} else {
    module.exports = {
        "presets": ["es2015", "react", "stage-0"]
    };
}